Libraries/Components/TextInput/TextInput.js

```diff
@@ -961,6 +961,15 @@ function InternalTextInput(props: Props): React.Node {
     return TextInputState.currentlyFocusedInput() === inputRef.current;
   }
 
+  function focus(): void {
+    const {current} = inputRef;
+    if (props.editable === false || current === null) {
+      return;
+    }
+    // $FlowFixMe - `focus` is missing in `$ReadOnly`
+    Object.getPrototypeOf(current)?.focus?.call(current);
```

Comment:
this links to https://github.com/facebook/react-native/blob/0.64-stable/Libraries/Renderer/implementations/ReactNativeRenderer-dev.js#L3895

without using Object.getPrototypeOf, we'd end up in a recursive call

Reply:
@yungsters had some feedback:
I think a better fix would be for the platform to return a boolean — true if the focus is successful, and false otherwise.

But for now, this logic should live in [TextInputState.focusTextInput](https://github.com/facebook/react-native/blob/main/Libraries/Components/TextInput/TextInputState.js#L89). 

```
function focusTextInput(textField: ?ComponentRef) {
  // …

  if (currentlyFocusedInputRef !== textField && textField != null) {
    // EXPLANATION…
    if (textField.props?.editable === false) {
      return;
    }

    // …
  }
}
```

Reply:
thanks